src/cycles/add-cycles-back.ts

```diff
@@ -0,0 +1,79 @@
+import * as _ from 'lodash';
+import { Edges, Graph, splitEdge } from './common';
+import { DepGraph, DepGraphBuilder, DepGraphData } from '..';
+import { GraphNode } from '../core/types';
+import { eventLoopSpinner } from 'event-loop-spinner';
+
+export async function addCyclesBack(depGraph: DepGraph) {
+  const depGraphData: DepGraphData = depGraph.toJSON();
+  const nodes: GraphNode[] = depGraphData.graph.nodes;
+  const edges: Graph = _(nodes)
+    .keyBy('nodeId')
+    .mapValues(({ deps, info }) => ({ info, deps: deps.map((y) => y.nodeId) }))
+    .value();
+  const pkgsInfoById = _(depGraphData.pkgs)
+    .keyBy('id')
+    .mapValues('info')
+    .value();
+
+  const removedEdgesToAdd: Edges = {};
+
+  const rootPkg = depGraph.rootPkg;
+  const depGraphBuilder = new DepGraphBuilder(depGraph.pkgManager, rootPkg);
+
+  for (const { nodeId, info, pkgId } of nodes) {
+    if (nodeId === 'root-node') continue;
+    let nodeInfo = info;
+    if (info?.labels && info?.labels['removed-cyclic-deps']) {
+      const cyclicEdges = info.labels['removed-cyclic-deps'].split(',').sort();
```

Comment:
Why the `sort()`?

Reply:
Example: array a=[1,2,3,4], if I take only odds numbers out, a=[2,4] and edges=[{originalIndex: 0, value: 1}, {originalIndex: 2, value: 3}]
If I'll insert the second one (value: 3) first, the array will [1,2,4,3].